app/controllers/api/v1/rooms_controller.rb

```diff
@@ -5,11 +5,10 @@ class RoomsController < Api::BaseController
       respond_to :json
 
       def index
-        if params[:conference_id].blank?
-          rooms = Room.all
+        if @conference
+          @conference.venue ? (rooms = @conference.venue.rooms) : (rooms = [])
```

Comment:
Hm, I don't like the rubocop suggestions ...
Do we need the rooms variable at all?

What about?
```
if @conference
  respond_with @conference.venue ? @conference.venue.rooms : Room.none
else
  respond_with Room.all
end

(note that this is untested...)
```